src/NHibernate.Test/Linq/PagingTests.cs

```diff
@@ -386,6 +386,28 @@ public void PagedProductsWithOuterWhereClause()
 			Assert.That(ids, Is.EqualTo(inMemoryIds));
 		}
 
+		[Test]
+		public void PagedProductsWithOuterWhereClause2()
```

Comment:
It adds no value compared to `PagedProductsWithOuterWhereClause`. Better to remove it.